### ISSUE TITLE
The build process is being killed on M2 processor

### ISSUE THREAD

**Issue by bvdmitri** [openjournals/inara#30]
I have the following script to compile my paper:
```bash
→ cat compile.sh
docker run --rm \
    --volume $PWD:/data \
    --user $(id -u):$(id -g) \
    --env JOURNAL=joss \
    openjournals/inara
```

Somewhere in the middle of the process I get "Killed" message. 
In the beginning docker complains with the following warning:
`
WARNING: The requested image's platform (linux/amd64) does not match the detected host platform (linux/arm64/v8) and no specific platform was requested
`

I tried to set `--platform linux/arm64/v8`, but it seems that there is no such image available.

**Comment by tarleb** [comment]
This is an unfortunate consequence of pandoc/dockerfiles#134. We built on those images, so there is nothing we can do until that issue is done.

**Comment by rikhuijzer** [comment]
I'm on a M1 and also got the "Killed" response. After following the process with
```sh
$ docker stats
```
it appears that the process runs out of memory during document building when using the default Docker settings. On my Mac, the default Docker memory limit was 7.9 GB and the process requires 12.94 GB of RAM during the execution of the `self-citation.lua` filter.

After bumping the default limit from 8 GB to 14 GB, the process does finish and takes about 3 minutes. On my x86 Linux it took only about 30 seconds. So it's about 5 times is slow while this processor is in general much quicker than my x86 was.

In summary, it does work when setting the memory limits to at least 13 GB, but it does run very inefficient.

**Comment by cthoyt** [comment]
FYI, pandoc 3.5 now supports multi-architecture builds!

**Comment by tarleb** [comment]
I'm considering a switch to `pandoc/latex:3.5-ubuntu` as the base image. That would add a few megabytes (~30MB), but would allow to build multi-platform images.
